src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,143 @@
-import React from 'react';
+import React, { useState, useCallback } from 'react';
 import './FindMovie.scss';
 
+import PropTypes from 'prop-types';
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+import { getMovies } from '../../api/api';
+import { Loader } from '../Loader/Loader';
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie = (
+  {
+    addMovieHandler,
+    isDuplicate,
+    setIsDuplicate,
+    isAdd,
+    setIsAdd,
+  },
+) => {
+  const [title, setTitle] = useState('');
+  const [foundMovie, setFoundMovie] = useState(null);
+  const [isFound, setIsFound] = useState(false);
+  const [isLoad, setIsLoad] = useState(false);
+
+  const findMovieHandler = useCallback(() => {
+    setIsLoad(true);
+    getMovies(title).then((response) => {
```

Comment:
```suggestion
    getMovie(title).then((response) => {
```
ты достаешь один фильм за раз